### PR TITLE
Development

### PR DESCRIPTION
**Pull request by garciajordy**
This is the first milestone of "TheNextWeb" :
- Made the layout responsive for mobile, tablet, and desktop.
- Add different navigation bars depending on the screen size.
- The project includes the header, footer, highlights, and latest funding.

### CODE REVIEW

**Comment by mimipeshy** [comment]
### STATUS: CHANGES REQUIRED 🔴 ♻️ 

Hello team @garciajordy and @Shaher-11 👋🏾 👋🏾 , great job so far in this milestone. I like how your website design looks, good job on the README file as well 😄 , however, a few things need to be fixed before its marked as complete

### Implementation details

* [x] CSS styles should use [media queries](https://www.w3schools.com/css/css_rwd_mediaqueries.asp) for 2 breakpoints
  * No small break point as it is mobile first and the base styles are for the mobile screen
  * medium: min-width around 768px
  * large: min-width 1024px
 

### Header

#### All screen sizes

- [x] Use the `header` tag for the header. This is the preferred [semantic](https://www.w3schools.com/html/html5_semantic_elements.asp) tag for this section. It should wrap the menus.
- [x] The logo should be  a link with (`a` tag)
- [x] Use links (`a` tag) for all the menu options, the hamburger button, and the search button


### Highlights (top 3 articles)
#### All screen sizes

* [x] The title of each article is a header (`hx` tag) and has a link (`a` tag)

#### Desktop

* [x] The TWN logo appears over the first article, at the top left corner
* [x] The date appears over the first article, at the top right corner

### Latest Funding section

#### All screen sizes

* [x] The title of the section is a header (`hx` tag) and has a link (`a` tag)
* [x] The nav menu (See more, Get updates, etc.) are grouped in a `nav` tag and each one have links (`a` tag)
* [x] Each article has
  * [x] A text excerpt at the bottom, that also has a link (`a` tag)

### Footer

* [x] It contains social media icons centered in a `nav` tag, each of them with a link (`a` tag)
* [x] Below the social media menu, there is another menu also centered, with a `nav` tag and each option with a link (`a` tag)


That's all :smile: kindly submit another review when done.

_Feel free to reach out to me on slack if you have any questions or something is unclear, or just comment on this PR and tag my name and I will be glad to help_
Regards
Peris Ndanu

**Comment by Shaher-11** [comment]
Done all the requested changes

File: index.html [hunk 1]
```diff
@@ -7,12 +7,611 @@
...
+</header>
+  <!-- #THE HEAD SECTION-->
+
+  <section class="head">
+
+    <div class="left-head top-right">
+      <h2 class="tnw">TNW</h2>
+      <a href="#"><span class="tnw-img"></span></a> 
+      <p class="date"> Tuesday -- January 26, 2021</p>
+      <a href="#" class="tnw-bottom">Opal cuts off your apps’ internet access to help you stop doomscrolling</a>
```
Comment (addod19) [inline index.html]: Kindly wrap all titles with `<h2>/<h3>` tags then wrap them with the `<a>` tag as in currently

File: index.html [hunk 2]
```diff
@@ -7,12 +7,611 @@
...
+      <a href="#" class="author">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago 12 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+    <div class="top-right-head left-head">
+      <a href="#" class="tnw-bottom">Google unveils privacy-friendly AI cookie killer</a>
```
Comment (addod19) [inline index.html]: Kindly wrap all titles with `<h2>/<h3>` tags then wrap them with the `<a>` tag as in currently

File: index.html [hunk 3]
```diff
@@ -7,12 +7,611 @@
...
+      <a href="#" class="author">by Tomas David <i class="far fa-clock"></i> 40 minutes ago 8 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+    <div class="bottom-right-head left-head">
+      <a href="#" class="tnw-bottom">This Mac app automatically detects your duplicate files</a>
```
Comment (addod19) [inline index.html]: Kindly wrap all titles with `<h2>/<h3>` tags then wrap them with the `<a>` tag as in currently

File: index.html [hunk 4]
```diff
@@ -7,12 +7,611 @@
...
+      </div>
+      <div class="latest3">
+        <a href="#">Uber expands its low-emission ride options to 1,400 more North American cities </a>
+        <p class="author"><a href="#">Matthew Beedham</a>22 minutes ago</p>
+      </div>
+      <div class="latest4">
+        <a href="#">3 reasons why almost every VC investor passed on Zoom </a>
+        <p class="author"><a href="#">Matthew Beedham</a>22 minutes ago</p>
+      </div>
+      <div class="latest5">
```
Comment (addod19) [inline index.html]: Kindly remove all commented code, use git history to retrieve previous code

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes:recycle: :traffic_light: 

Hi @garciajordy and @Shaher-11, you guys did a great job working on the comments by the previous TSE :+1: Most of the comments have been addressed, however, I would like to highlight a few we need to work on. Kindly check the comments below: :fireworks: 

- The "TNW" logo must appear on a tablet/medium screen size. But currently, it is not there. Let us also fix the text in the "white" background :slightly_smiling_face: 
`Original`
no
`Clone`
no1
- There is a huge margin between the "Highlights" section and the "Latest funding" section. Let us kindly fix it
- KIndly center and fix the footer section as in the screenshot below
`Original`
f
`Clone`
f1

Once these are fixed, kindly submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by garciajordy** [comment]
Done the requested changes
We put the margin before as it looks like the original website however we changed everything as requested

File: index.html [hunk 5]
```diff
@@ -7,12 +7,622 @@
...
+        <p class="author"><a href="#">Matthew Beedham</a>22 minutes ago</p>
+      </div>
+      <div class="latest1">
+        <a href="#">Signal is drama-free for now, but it should prepare for the worst </a>
+        <p class="author"><a href="#">Matthew Beedham</a>22 minutes ago</p>
+      </div>
+    </div>
+
+  </section>
+-->
```
Comment (bini-i) [inline index.html:190]: Kindly remove comments from committed code.

Comment (bini-i) [inline index.html]: Kindly remove comments from committed code.

File: index.html [hunk 6]
```diff
@@ -7,12 +7,622 @@
...
+          <li><a href="#">Winklevoss comes out of meditation to witness Bitcoin's record high</a></li>
+          <li><a href="#">CD Projekt Rekt: Cyberpunk 2077 tanks studio's share price nearly 50%</a></li>
+          <li><a href="#">CD Projekt stock down 20% since Cyberpunk reviews hit the internet</a></li>
+          <li><a href="#">Airbnb to be bigger than eBay after its long-awaited IPO</a></li>
+        </ul>
+      </div>
+    </div>
+
+  </section>
+-->
```
Comment (bini-i) [inline index.html]: Kindly remove comments from committed code.

File: index.html [hunk 7]
```diff
@@ -7,12 +7,622 @@
...
+        <a href="#">The 2021 Complete Microsoft Azure Certification Prep Bundle </a>
+        <p class="author"><abbr>$1345</abbr> $39.90</p>
+      </div>
+      <div class="latest1">
+        <a href="#">Familio Private Backup Platform: Lifetime Subscription</a>
+        <p class="author"><abbr>$1345</abbr> $39.90</p>
+      </div>
+    </div>
+
+  </section>
```
Comment (bini-i) [inline index.html]: Kindly remove comments from committed code.

File: index.html [hunk 8]
```diff
@@ -7,12 +7,622 @@
...
+      <h2 class="tnw">TNW</h2>
+      <a href="#"><span class="tnw-img"></span></a>
+      <p class="date"> Tuesday -- January 26, 2021</p>
+      <a href="#" class="tnw-bottom">Opal cuts off your apps’ internet access to help you stop doomscrolling</a>
+      <a href="#" class="author">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago 12 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+    <div class="top-right-head left-head">
+      <a href="#" class="tnw-bottom">Google unveils privacy-friendly AI cookie killer</a>
```
Comment (bini-i) [inline index.html]: kindly put titles in header tags (h2, h3, ...) to semantically identify them as titles

File: index.html [hunk 9]
```diff
@@ -7,12 +7,622 @@
...
+      <a href="#" class="author">by Tomas David <i class="far fa-clock"></i> 40 minutes ago 8 <i
+          class="fas fa-share-alt"></i></a>
+    </div>
+
+    <div class="bottom-right-head left-head">
+      <a href="#" class="tnw-bottom">This Mac app automatically detects your duplicate files</a>
```
Comment (bini-i) [inline index.html]: kindly put titles in header tags (h2, h3, ...) to semantically identify them as titles

File: index.html [hunk 10]
```diff
@@ -7,12 +7,622 @@
...
+  </header>
+  <!-- #THE HEAD SECTION-->
+
+  <section class="head">
+
+    <div class="left-head top-right">
+      <h2 class="tnw">TNW</h2>
+      <a href="#"><span class="tnw-img"></span></a>
+      <p class="date"> Tuesday -- January 26, 2021</p>
+      <a href="#" class="tnw-bottom">Opal cuts off your apps’ internet access to help you stop doomscrolling</a>
```
Comment (bini-i) [inline index.html]: kindly put titles in header tags (h2, h3, ...) to semantically identify them as titles

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Changes Required :yellow_circle: 
### Hello  @Shaher-11 and @garciajordy :wave: 

### Thanks for working on the changes suggested. You have done a great job and your project passes most of the requirements, but you have to work on all requirements suggested by previous TSE for this project to be approved. 

#### Please check comments below :point_down: and work on them before requesting a new code review

### Suggestions
- Please resize footer elements in mobile view as they are overflowing the screen currently.
image

#### If you have any question kindly reach me on slack @Binyam Hailemeskel

### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (TSE)
> GitHub: [@bini-i](https://github.com/bini-i)
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

File: index.html (hunk 4 above)
Comment (Shaher-11) [reply index.html]: The commented code is for the other sections of the website as we have structured the website and styled the shown sections as requested

**Comment by Shaher-11** [comment]
Done the requested changes

**Review by smcommits — CHANGES_REQUESTED** [review]
# Status: Changes Required
Hello, team.
**Great job! Your webpage looks great. 
You have implemented almost all of the changes required but you still have one left.** 

## Required Changes

You still haven't implemented the following change that was required by the previous TSE.

- [ ]  Make the footer nav responsive

Your footer nav is still not responsive and is overflowing out of the screen. The project is about making responsive web pages. Therefore your elements must be responsive. 

cr

****Hint**: You have ul and div elements in your footer nav which have the property of `display: flex`.  Try to find out which flex property helps you to wrap the contents into the next line once there is not enough horizontal space available in the container.** 

Please submit a code review request after implementing the changes.

# 
If you need any help or requrie any further clarifications, please reach out to me on [slack](https://microverse-students.slack.com/team/U017N7LLZ40).

**Comment by Shaher-11** [comment]
Done All the requested changes and made the footer completely responsive and an exact version of the original website

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔✔✔**

Hello @garciajordy ,
Great job implementing the required changes. You can merge the branches
See you in the next project👏👌

**Pull request merged**